Auto-ID manuals the player steps on

isloat opened the pull request:
Having to pick up the manual to see what it'd train was annoying (especially with a full inventory) and
inconsistent with current book behavior.

Since you have to actually reach the tile that the manual resides on, the 'mystery' is still preserved.

PleasingFungus commented:
What's the messaging like here? Do you get informed about whatever you just stepped on?

isloat commented:
(xv view) A paperback manual.
(after stepping) You see here a manual of Fighting.

MarvinPA commented:
As far as I can tell, this works for the first manual in a given game and then stops working for any subsequent manuals (they're considered fully identified already?).

isloat commented:
I've fixed the bug MarvinPA mentions and squashed it into the original commit. 

As far as I could tell, the bug happened because of line 943, where it checked if books have been fully identified. As far as I can tell there's no static list of manual types, so it saw that we'd identified one manual (out of one) and then returned out of the function. Adding an exception for manuals gets us intended behavior.
